thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -11,11 +11,19 @@ include "../shared.thrift"
 enum RelatedStoryLinkType {
   TAG = 0,
   STORY = 1,
+  CONTENT = 2,
+  ATOM = 3
+}
+
+struct Answer {
+  1: required string relatedStoryId
+  2: required RelatedStoryLinkType relatedStoryLinkType
```

Comment:
Are we overloading the meaning of `relatedStory` here? I think the original intention of the top-level `relatedStoryId`/`relatedStoryLinkType` was to attach these questions to a 'story', rather than an answer?

I don't think you'd ever answer a question with a story or tag, since these are used to group content together.
It would be less confusing to have something like:
```
enum AnswerType {
  CONTENT = 0,
  ATOM = 1
}

struct Answer {
  1: required string answerId
  2: required AnswerType answerType
}
```

Reply:
@tomrf1 I think you are right.I will update the pull request.